docs/devel/coding-conventions.md

```diff
@@ -33,60 +33,123 @@ Documentation for other releases can be found at
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 Code conventions
   - Bash
+
     - https://google-styleguide.googlecode.com/svn/trunk/shell.xml
-    - Ensure that build, release, test, and cluster-management scripts run on OS X
+
+    - Ensure that build, release, test, and cluster-management scripts run on
+OS X
```

Comment:
I presume the two important ones are the linux man(ual) which are the cli text manuals and man pages which is being used by GitHub.  I see there has been an effort to compile the kubectl-*.md docs into a linux manual.